### DISCUSSION TITLE
Weird indentation in multiline code blocks

### DISCUSSION THREAD

**Discussion by joeltine** [requarks/wiki#4850] [Error / Bug Report] [answered]
I noticed the indentation is off on the first indented line of a multiline code block. It looks fine when you are editing/previewing the markdown, but once you save the page, it indents incorrectly. This is Wiki.js v2.5.268.

For example, here's a screenshot where the markdown and rendered  preview look fine:

<img width="1032" alt="Screen Shot 2021-12-31 at 9 58 44 PM" src="https://user-images.githubusercontent.com/13138272/147844300-64c16c43-b094-41f8-9452-68f8a14f2de1.png">

This is after saving the page, note the incorrect indentation on the `-e foo=bar` line:

<img width="443" alt="Screen Shot 2021-12-31 at 9 58 57 PM" src="https://user-images.githubusercontent.com/13138272/147844298-b3b4e9ad-3147-4f74-b83b-b3c1f67e385a.png">

**Comment by NGPixel** [comment, marked as answer]
Make sure you're not using tabs in that second line.

**Reply by joeltine** [reply]
Indeed, it was a tab not spaces. Changing to spaces fixed it.
